kafka-connector/src/main/java/com/google/pubsub/kafka/source/CloudPubSubGRPCSubscriber.java

```diff
@@ -39,9 +39,11 @@
   private long nextSubscriberResetTime = 0;
   private SubscriberFutureStub subscriber;
   private Random rand = new Random(System.currentTimeMillis());
+  private String gcpCredentialsFilePath;
 
-  CloudPubSubGRPCSubscriber() {
+  CloudPubSubGRPCSubscriber(String gcpCredentialsFilePath) {
```

Comment:
We should make the same changes on the publisher side so one can specify credentials for the sink connector, too.